dockstore-webservice/src/main/java/io/dockstore/webservice/core/AppTool.java

```diff
@@ -18,11 +18,16 @@
 import io.dockstore.common.EntryType;
 import io.swagger.annotations.ApiModel;
 import javax.persistence.Entity;
+import javax.persistence.NamedQueries;
+import javax.persistence.NamedQuery;
 import javax.persistence.Table;
 
 @ApiModel(value = "AppTool", description = "This describes one app tool in dockstore as a special degenerate case of a workflow", parent = Workflow.class)
 @Entity
 @Table(name = "apptool")
+@NamedQueries({
+    @NamedQuery(name = "io.dockstore.webservice.core.AppTool.findAllPublished", query = "SELECT a FROM AppTool a WHERE a.isPublished = true")
```

Comment:
Absurdly small pet peeve: 
Think you can omit the `= true` 
At least in regular SQL you can do 
```
webservice_test=# select * from enduser where not isadmin;
webservice_test=# select * from enduser where isadmin;

```

Reply:
Keeping because most of queries are written this way.